lib/src/components/Banner.tsx

```diff
@@ -9,13 +9,71 @@ type TBanner = {
 	image?: string,
 	primaryButton?: ReactElement,
 	secondaryButton?: ReactElement,
-	children: ReactElement | ReactElement[],
+	children?: ReactElement | ReactElement[],
 	canClose?: boolean,
 	onClose?: () => void,
 	onClick?: () => void,
-	maxHeight?: string,
 	variant?: 'default' | 'image' | 'split' | 'background'
 }
+type TDefaultVariant = {
+	title?: string,
+	primaryButton?: ReactElement,
+	secondaryButton?: ReactElement,
+	children?: ReactElement | ReactElement[],
+	variant?: 'default' | 'image' | 'split' | 'background'
+}
+
+function	SplitVariant({image}: {image: string}) {
+	return (
+		<div className={'w-full md:w-1/2 relative'}>
+			<div
+				className={'flex md:hidden overflow-hidden relative -mx-1 -mt-1 w-full h-full rounded-xl border-x-2 border-b-2 border-primary max-h-48'}
+				style={{width: 'calc(100% + 8px)'}}>
+				<div className={'rounded-lg image-align-middle'}>
+					<img src={image} className={'w-full h-full object-cover'} loading={'eager'} />
+				</div>
+			</div>
+			<div
+				className={'hidden md:flex overflow-hidden -my-1 -ml-1 w-full h-full rounded-xl border-y-2 border-l-2 border-primary absolute'}
+				style={{height: 'calc(100% + 8px)', width: 'calc(100% + 8px)'}}>
```

Comment:
`-my-1 -ml-1` and `height: 'calc(100% + 8px)', width: 'calc(100% + 8px)'` are used to fit the image with the borders and avoid a few pixel spaces